pcdsdevices/lens.py

```diff
@@ -286,31 +341,294 @@ def _make_safe(self):
             filt.insert()
             time.sleep(0.01)
         if filt.inserted:
-            print("REMINDER: Beam stop attenuator moved in!")
```

Comment:
Do we want this as `prints` instead of loggings - i think there might be a reason for it, if so i'll change it back?

Reply:
We almost always want logging. Only potential exception in my mind is if it's _solely_ a high-level function displaying interactive information to the user that we don't care to have logged.

Reply:
Logging is better here

Reply:
This `print` is gone but a few other `print()` statements seem to remain in error

Reply:
Ok thanks Ken. 
The only one that's left now is the one from here:
https://github.com/pcdshub/pcdsdevices/blob/f771f2a1fbc4b3a1daa3dc48d1bccb3e47fdcf8f/pcdsdevices/lens.py#L254-L260

I don't think we mean to log it though - should I still make it a `logger.debug()` or something like this?